pkg/aws/commandbuilder/helper/roles/roles.go

```diff
@@ -27,6 +28,9 @@ func ManualCommandsForMissingOperatorRole(input ManualCommandsForMissingOperator
 		tags.OperatorName:      input.Operator.Name(),
 		tags.RedHatManaged:     "true",
 	}
+	if input.ManagedPolicies {
+		iamTags[tags.ManagedPolicies] = "true"
```

Comment:
I would love replace the `if` something like 
```
iamTags[tags.ManagedPolicies] = fmt.Sprintf("%v", input.ManagedPolicies)
```

But we probably don't want the tag if it's false, right?

Reply:
Yes, since all existing roles are non-managed and don't have the tag.
To be consistent with existing roles.
Otherwise, we'll have to check later if the tags exist and their values, currently, if the tag exists it's managed.